presto-tests/src/test/java/com/facebook/presto/tests/TestMetadataManager.java

```diff
@@ -27,6 +27,7 @@
  * while registering catalog -> query Id mapping.
  * This mapping has to be manually cleaned when query finishes execution (Metadata#cleanupQuery method).
  */
+@Test(sequential = true)
```

Comment:
It is in the commit message.
Do you think we should have this as a comment in the code as well? I feel like commit msg is sufficient but if you are insistive I can add it here as well. :)